admin/services/config.service.js

```diff
@@ -67,8 +71,14 @@ const configService = {
       if (code === accessArrangementsDataService.CODES.AUDIBLE_SOUNDS) checkOptions.audibleSounds = true
       if (code === accessArrangementsDataService.CODES.INPUT_ASSISTANCE) checkOptions.inputAssistance = true
       if (code === accessArrangementsDataService.CODES.NUMPAD_REMOVAL) checkOptions.numpadRemoval = true
-      if (code === accessArrangementsDataService.CODES.FONT_SIZE) checkOptions.fontSize = true
-      if (code === accessArrangementsDataService.CODES.COLOUR_CONTRAST) checkOptions.colourContrast = true
+      if (code === accessArrangementsDataService.CODES.FONT_SIZE) {
+        checkOptions.fontSize = true
```

Comment:
Do we really need a flag and a code?   If we must have both then the flag may be better called something like `hasCustomFontSize`

Reply:
@jon-shipley this is not a flag for the code really but instead to identify that the font size access arrangements is activated.
There are 2 different cases here:

1) Font size access arrangement is activated (`checkOptions.fontSize = true`) but not pupil font size preference has been set (`fontSizeCode` will be `null`)

2) Font size access arrangement is activated and the pupil chose a font size preference which got stored in the db. In that case when a pin is being generated again it will include the pupil font size preference as well (checkOptions.fontSizeCode='VHS'`)